trainingRN/src/screens/todo/index.js

```diff
@@ -35,13 +38,25 @@ class TodoContainer extends Component {
     this.setState(prevState => ({ todoList: [...prevState.todoList, newTask] }))
   }
 
+  handleDeleteAll = () => {
+    this.setState({ todoList: [] })
+  }
+
+  checkElem = id => {
+    console.log(id);
+    const newList = this.state.todoList.map(elem => elem.id === id ? {...elem, checked: !elem.checked} : elem);
+    this.setState({ todoList: newList });
+  }
+
   render() {
     return (
     <TodoLayout
       handleDelete={this.handleDelete}
```

Comment:
This should be
```
onDelete={this.handleDelete}
```
Some of this is being explained [here](https://jaketrent.com/post/naming-event-handlers-react/)